packages/flutter/test/material/app_bar_test.dart

```diff
@@ -10,7 +10,7 @@ import 'package:flutter_test/flutter_test.dart';
 
 import '../widgets/semantics_tester.dart';
 
-Widget buildSliverAppBarApp({ bool floating, bool pinned, double expandedHeight, bool snap = false }) {
+Widget buildSliverAppBarApp({ bool floating, bool pinned, double collapsedHeight, double expandedHeight, bool snap = false }) {
```

Comment:
Nit: Parameter list is getting long, add a trailing comma and reformat it

Reply:
Done!